CONTRIBUTING.md

```diff
@@ -291,6 +291,10 @@ Current Dockerfiles
   - installs Secureli
   - Checks out the public pip repo, inits secureli into the repo and runs a scan
 
+## Best Practices
```

Comment:
This was my best guess as to how to include the Pythonic coding patterns specified in the ticket. Open to suggestions on if we want this done differently.

Reply:
This is great. That was admittedly an ambiguous AC